input.go

```diff
@@ -52,6 +52,11 @@ func CursorPosition() (x, y int) {
 	return ui.AdjustedCursorPosition()
 }
 
+// MouseScroll returns the y offset of the mouse wheel
```

Comment:
From the comment, I couldn't understand when this function return 0 or non-0 value.

Reply:
BTW, is Y axis enough? Doesn't wheeling happen both on X an Y axes?

Reply:
I see your point. I oriented that comment on the variable names in the callback (xoff, yoff). In my mind it works that way that the offset of a normal mouse wheel is always set back to 0 when it's not used. I'll add that.

For me, Y axis was enough, but i guess I should add it while I'm at it.